modules/cnv-delete-vm-web.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-delete-vms.adoc
+
+[id="cnv-delete-vm-web_{context}"]
+
+= Deleting a virtual machine using the web console
+
+Deleting a virtual machine permanently removes it from the cluster.
+
+Delete a virtual machine using the &#8942; button of the virtual machine in the *Workloads* -> *Virtual Machines* list, or using the *Actions* control of the *Virtual Machine Details* screen.
+
+.Procedure
+
+. Click *Workloads* -> *Virtual Machines* from the side menu.
```

Comment:
```
. In the {ProductName} console, click *Workloads* -> *Virtual Machines*.
```

Reply:
Changed to:

". In the {ProductName} console, click *Workloads* -> *Virtual Machines* from the side menu."